### PR TITLE
[fix](nereids) Fix add column syntax

### PR DESCRIPTION
**Pull request by w41ter**
The original parser allows the column_definition_list only to have one column_definition, this PR makes the nereids parser compatible with the legacy one.

### CODE REVIEW

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by gavinchou — DISMISSED** [review]

**Comment by w41ter** [comment]
run buildall

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4 [hunk 1]
```diff
@@ -237,7 +237,7 @@ addRollupClause
 
 alterTableClause
     : ADD COLUMN columnDef columnPosition? toRollup? properties=propertyClause?     #addColumnClause
-    | ADD COLUMN LEFT_PAREN columnDef (COMMA columnDef) RIGHT_PAREN
+    | ADD COLUMN LEFT_PAREN columnDef (COMMA columnDef)? RIGHT_PAREN
```
Comment (morrySnow) [inline fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4:240]:
```suggestion
    | ADD COLUMN LEFT_PAREN columnDef (COMMA columnDef)* RIGHT_PAREN
```

**Comment by w41ter** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by gavinchou — APPROVED** [review]

**Pull request merged**